src/task_impl/mod.rs

```diff
@@ -242,15 +242,17 @@ impl<T> Spawn<T> {
     /// Get a mutable reference to the object the Spawn is wrapping.
     pub fn get_mut(&mut self) -> &mut T {
         &mut self.obj
-    }
+    } 
+}
 
+impl<T> Spawn<T> {
```

Comment:
Mind leaving these in the same impl block, but adding `where Self: Sized` to this method?

Reply:
If you're super enterprising, you could collapse all the below impl blocks into one with the same strategy as well :)

(no need to happen in this PR though)